Source/Core/VideoCommon/Fifo.cpp

```diff
@@ -391,9 +388,6 @@ void FifoManager::RunGpuLoop()
         }
       },
       100);
-
-  AsyncRequests::GetInstance()->SetEnable(false);
```

Comment:
This was flushing the async request queue.  I'm guessing this isn't needed any more because by joining the GPU thread, the remaining requests will flow out naturally.  Is that correct?

Reply:
Yeah. This seems more sensible to me instead of just cutting off GPU processing before the CPU stops.